File: Arrays/1_introduction.py
array = [1,2,3,4,5,6,7]
print(array)

# Insert at End O(1)
array.append(8)
print(array)

# Delete from End O(1)
array.pop()
print(array)

# Insert at Beginning O(n)
array.insert(0,0)
print(array)

# Insert at index 3 O(n)
array.insert(3,9)
print(array)

# Remove from index 3 O(n) 
array.pop(3)
print(array)

File: Arrays/1_introduction.js
var character_array = ['a','b','c','d','e']

character_array.push('f'); // O(1)
console.log("Insert at End");
console.log(character_array);

character_array.pop()  // O(1)
console.log("\nDelete at End");
console.log(character_array);

character_array.unshift('o');  // O(n)
console.log("\nInsert at Beginning");
console.log(character_array);

character_array.splice(2,0,'z'); //O(n)
console.log("\nInsert at index 2");
console.log(character_array);

character_array.splice(2,2); //O(n)
console.log("\nDelete 2 elements from index 2");
console.log(character_array);